schemas/alpha/azure.yaml.json

```diff
@@ -504,6 +505,20 @@
                     "examples": [
                         ">= 0.6.0-beta.3"
                     ]
+                },
+                "extensions": {
+                    "type": "object",
+                    "title": "A range of supported versions of extensions for this project",
```

Comment:
Wondering if we should mention that whatever extension is listed here is a `must have` for init and use the template.  

For example, failing to install the extension blocks azd from initializing the template.  The extension is not really optional when it is listed here because the functionality of the template depends on the extension's existence.

Or, should we make `extensions` an object array where, for each extension you can define more things for the extension?  Like the version, if it is mandatory for the template or optional, and any other future setting?

Reply:
Agreed that if you specify it here that it should be considered required otherwise you're not going to be able to get the full benefit of the template without it.